actionsystemtest/lib/action_system_test/test_helpers.rb

```diff
@@ -0,0 +1,9 @@
+module ActionSystemTest
+  module TestHelpers
+    extend ActiveSupport::Autoload
+
+    autoload :ActiveJobSetup
```

Comment:
Isn't `autoload` generally discouraged? Maybe this was fixed in newer Ruby versions?